### PR TITLE
revendor (org.opencontainers.ref.name -> org.opencontainers.image.ref…

### PR DESCRIPTION
**Pull request by AkihiroSuda**
….name)

Fix https://github.com/projectatomic/skopeo/issues/348

https://github.com/containers/image/pull/278#issuecomment-304257815

Signed-off-by: Akihiro Suda <suda.akihiro@lab.ntt.co.jp>

### CODE REVIEW

File: vendor.conf [hunk 1]
```diff
@@ -45,3 +45,5 @@ github.com/pborman/uuid v1.0
 github.com/opencontainers/selinux master
 golang.org/x/sys/unix master
 github.com/tchap/go-patricia v2.2.6
+github.com/BurntSushi/toml b26d9c308763d68093482582cea63d69be07a0f0
+
```
Comment (runcom) [inline vendor.conf]: drop newline

File: vendor.conf [hunk 2]
```diff
@@ -1,8 +1,8 @@
 github.com/urfave/cli v1.17.0
-github.com/containers/image master
+github.com/containers/image 84d3787ee366601bdb0a31ca26261c20a0a60e64
```
Comment (runcom) [inline vendor.conf]: leave this as master please

Comment (AkihiroSuda) [inline vendor.conf]: Do I need to use master for storage as well?

**Comment by runcom** [comment]
LGTM

File: vendor.conf (hunk 2 above)
Comment (runcom) [inline vendor.conf]: Nope, fixed c/storage is OK

**Comment by AkihiroSuda** [comment]
reran `vndr`. some lines are updated by `vndr`

**Comment by runcom** [comment]
@AkihiroSuda after you edited `vndr.conf` you should have just ran:

```
vndr github.com/containers/image
vndr github.com/BurntSushi/toml
```

Please do that and discard any other fixes/changes you did, let's make a separate PR for that.

**Comment by mtrmac** [comment]
> @AkihiroSuda after you edited `vndr.conf` you should have just ran:
> 
> ```
> vndr github.com/containers/image
> vndr github.com/BurntSushi/toml
> ```
> 
> Please do that and discard any other fixes/changes you did, let's make a separate PR for that :heart: 

That is locally convenient, but it also makes it way too easy for `vendor.conf` and the contents of `vendor` to drift away.

I guess both local vendor updates like the above (_if_ the complete refresh happens frequently enough), and a full `vndr` re-run within an otherwise unrelated PR, are fine with me.

(<s>Non-ABI-stable dependencies and vendoring and staying on a specific commit without future bugfixes is all problematic, \*grumble\*</s> I don’t have a good solution given the existing ecosystem, \*shuts up\*.)

---

The current test failure looks like a missing `git add`.

**Comment by AkihiroSuda** [comment]
revendored

- I needed to use older vndr (LK4D4/vndr@78865), because the current vndr requires me to rewrite `vendor.conf` to the new format
- Eliminated vendoring for toml (unneeded for this case)

**Comment by mtrmac** [comment]
This updates `vendor.conf`to point to a newer containers/storage version, yet the PR is not actually updating containers/storage, and the dependency necessary for that new version is not listed either; that’s just a trap left for the next person to run `vndr`.

Please, either run `vndr` only `containers/image`, in which case leave the other repos in `vendor.conf` unmodified, or do a full `vndr` run where both the `vendor` directory and `vendor.conf` are fully consistent.

**Comment by AkihiroSuda** [comment]
vndred containers/image@master only, PTAL?

**Comment by runcom** [comment]
LGTM but needs rebase

**Comment by AkihiroSuda** [comment]
github says "Pull request successfully merged and closed" but it does not seems merged..

**Comment by runcom** [comment]
> github says "Pull request successfully merged and closed" but it does not seems merged..

:confused: yeah, it seems closed actually.

**Comment by runcom** [comment]
I can't reopen it :confused:

**Comment by AkihiroSuda** [comment]
np, https://github.com/projectatomic/skopeo/commit/63272a10d7f8116d06aca8324889ff1174db6f42 already contains this change

**Comment by AkihiroSuda** [comment]
and hence rebase & push -f resulted in closing this PR

**Pull request closed without merging**